src/funcTest/resources/java-project-with-feature-previews/build.gradle

```diff
@@ -25,16 +25,17 @@ repositories {
 // Configure Java compilation tasks to enable feature previews
 tasks.withType(JavaCompile).configureEach {
     options.compilerArgs.add("--enable-preview")
+    options.release = 22
```

Comment:
```suggestion
```

Reply:
[That's](https://scans.gradle.com/s/edpef2zzfn2va/tests/task/:functionalTest/details/me.champeau.jmh.ProjectWithFeaturePreviewSpec/successfully%20executes%20benchmark%20which%20uses%20feature%20previews%20(Gradle%207.0)?top-execution=1) the failure

Reply:
Should we pin toolchain to Java 21 in this test?

Reply:
Must be 22 - 21 doesn't allow enable-preview